app/views/browse/index.html.erb

```diff
@@ -0,0 +1,57 @@
+
+<h1>Browse by Call Number</h1>
+
+<h2 class="h4"><%= @shelf_list.list.first.call_number %> to <%= @shelf_list.list.last.call_number %></h2>
+
+<div class="row my-3">
+  <div class="col-6">
+    <%= render Browse::PageNavigation.new(prev_item: @shelf_list.previous,
+                                          next_item: @shelf_list.next,
+                                          length: params[:length]) %>
+  </div>
+  <div class="col-6 text-right">
+    <%= render Browse::PageSizeSelector.new(params: params) %>
+  </div>
+</div>
```

Comment:
@nicktiberi can we move lines 6-15 to a prev_next_component and render both here and at the bottom instead lines 48-57?

@awead

Reply:
One of is a page navigation component, and the other is a page size component. They are similar, but slightly different. Are you suggesting a third component that contains the two of them?

Reply:
Yeah, I was thinking of a third component that contains all the repeated lines. Ah, right just realized one of them is the size selector but regardless yeah just like doing a one render rather than two with the divs around it moved into a component as well? 🤔

Reply:
@banukutlu - makes sense to me! I'm finishing up a wrapper component and will commit in a bit here.

Reply:
kewl, thanks!!